pkg/operator/resources/cluster/apiserver.go

```diff
@@ -96,10 +97,10 @@ func getAPIServerClusterPolicyRules() []rbacv1.PolicyRule {
 				"cdi.kubevirt.io",
 			},
 			Resources: []string{
-				"*",
+				"datavolumes",
 			},
 			Verbs: []string{
-				"*",
+				"list",
 			},
```

Comment:
We intend to backport the commit to a release immediately, and I'm slightly concerned about unintended effects from this change. Perhaps we should have it as a separate change that isn't backported.

Reply:
Not too concerned about this as the cdi client was not even imported before this PR.